frontend/src/testing/pushapi-mock.js

```diff
@@ -1,28 +1,21 @@
 /**
- * @author Stefan Dimitrov (stefan.dimitrov@clouway.com).
- */
+  * @author Stefan Dimitrov (stefan.dimitrov@clouway.com).
+  */
+ angular.module('clouway-push', [])
 
-angular.module('clouway-push')
+   .config(function ($provide, pushApiProvider, $httpBackend) {
```

Comment:
Added this in the pushapi-mocks.js, but I'm not sure that I understand how this will help other users.